test/FluentMigrator.Tests/Unit/Initialization/ConventionSetTests.cs

```diff
@@ -67,12 +67,8 @@ public void NotLoadingConventionSetFromAssemblyWithScanForMigrations()
             Assert.AreEqual(null, expr.SchemaName);
         }
 
-        /// <summary>Demonstrate that AddFluentMigratorCore() always registers a class to handle
-        /// IConventionSet; even when one has already been registered in the services collection.
-        /// Because the ServiceCollection is LIFO, the last one registered is the one that gets
-        /// used.</summary>
```

Comment:
@tgharold This change looks good but did you mean to delete this comment?

Reply:
Yes, the first commit was to demonstrate failure.  Later commits made that comment obsolete.